.vscode/settings.json

```diff
@@ -12,7 +12,10 @@
         "bsp_timer.h": "c",
         "temperature.h": "c",
         "math.h": "c",
-        "bsp_adc.h": "c"
+        //"bsp_adc.h": "c",
+        "basp_adc.h": "c",
```

Comment:
Why did you do this? Did you manually edit this file?

Reply:
deleted it line 15